### PR TITLE
Remove deprecated web triggers

### PR DESCRIPTION
**Pull request by emmadickson**
Supports https://artsyproduct.atlassian.net/browse/PLATFORM-2688

### CODE REVIEW

File: src/desktop/apps/consign/client/analytics_middleware.ts [hunk 1]
```diff
@@ -21,8 +21,9 @@ const analyticsMiddleware = store => next => action => {
       })
       return result
     }
+
     case actions.SUBMISSION_CREATED: {
-      analyticsHooks.trigger("consignment:submitted", {
+      analytics.track("consignment_submitted", {
```
Comment (damassi) [inline src/desktop/apps/consign/client/analytics_middleware.ts:26]: Should this be `consignment:submitted`?  Noticing that there's some idiom mixing between `:` and `_` here and below.

Comment (eessex) [reply src/desktop/apps/consign/client/analytics_middleware.ts:26]: The old version is a mediator trigger, but we are firing the event directly and use the event name.

File: src/desktop/apps/consign/client/__tests__/analyticsMiddleware.jest.ts [hunk 2]
```diff
@@ -47,12 +48,12 @@ describe("analyticsMiddleware", () => {
...
 
-    expect(triggerStub.callCount).toBe(1)
-    const analyticsArgs = triggerStub.firstCall.args
-    expect(analyticsArgs[0]).toBe("consignment:submission:error")
-    expect(analyticsArgs[1]).toEqual({
-      type: "convection_create",
+    expect(window.analytics.track).toHaveBeenCalledTimes(1)
+    expect(
+      window.analytics.track
+    ).toBeCalledWith("consignment:submission:error", {
```
Comment (eessex) [inline src/desktop/apps/consign/client/__tests__/analyticsMiddleware.jest.ts:54]: @emmadickson @damassi sorry i did not read the comment close enough-- yes the use is inconsistent, we should only use `snake_case` naming for events (the old syntax is for mediator). This event will need its name changed.

File: src/desktop/apps/consign/client/__tests__/analyticsMiddleware.jest.ts [hunk 3]
```diff
@@ -97,10 +98,8 @@ describe("analyticsMiddleware", () => {
     })
     store.dispatch({ type: "SUBMISSION_COMPLETED" })
 
-    expect(triggerStub.callCount).toBe(1)
-    const analyticsArgs = triggerStub.firstCall.args
-    expect(analyticsArgs[0]).toBe("consignment:completed")
-    expect(analyticsArgs[1]).toEqual({
+    expect(window.analytics.track).toHaveBeenCalledTimes(1)
+    expect(window.analytics.track).toBeCalledWith("consignment:completed", {
```
Comment (eessex) [inline src/desktop/apps/consign/client/__tests__/analyticsMiddleware.jest.ts:102]: This naming is also incorrect.

**Pull request merged**